src/bucket.ts

```diff
@@ -616,6 +621,8 @@ class Bucket extends ServiceObject {
   getFilesStream: Function;
   signer?: URLSigner;
 
+  originalRetryValue?: boolean;
```

Comment:
Maybe rename this to `instanceRetryValue`? Makes it more apparent that we want to override the value set at the Storage constructor level.